security/certificate_types_descriptions/node-certificates.adoc

```diff
@@ -8,12 +8,31 @@ toc::[]
 
 == Purpose
 
-Node certificates are signed by the cluster; they come from a certificate authority (CA) that is generated by the bootstrap process. After the cluster is installed, the node certificates are auto-rotated.
+Node certificates are signed by the cluster and allow the kubelet to communicate with the Kubernetes API server. They come from the kubelet CA certificate, which is generated by the bootstrap process.
+
+== Location
+
+The kubelet CA certificate is located in the `kube-apiserver-to-kubelet-signer` secret in the `openshift-kube-apiserver-operator` namespace.
```

Comment:
So far as I know the kubelet CA certificate is located in the following path on the nodes.
```
sh-4.4# cat /etc/kubernetes/kubelet.conf
kind: KubeletConfiguration
apiVersion: kubelet.config.k8s.io/v1beta1
authentication:
  x509:
    clientCAFile: /etc/kubernetes/kubelet-ca.crt
```
Some of the opinions here need to be confirmed by @vrutkovs

Reply:
Both are correct - there is "CA for kube-apiserver to use when contacting kubelet for exec/logs" - that is `kube-apiserver-to-kubelet-signer`, see [this](https://github.com/openshift/api/blob/master/tls/docs/kube-apiserver%20Client%20Certificates/README.md#kube-apiserver-to-kubelet-signer). There is also "CA used by kubelet to contact kubeapiserver" - that is `admin-kubeconfig-signer` stored in `/etc/kubernetes/kubelet-ca.crt`, see [this](https://github.com/openshift/api/blob/master/tls/docs/kube-apiserver%20Client%20Certificates/README.md#admin-kubeconfig-signer).

I think its worth documenting both

Reply:
`admin-kubeconfig-signer` is generated on bootstrap, so its mentioned on bootstrap page